redwood-gradle-plugin/src/main/kotlin/app/cash/redwood/gradle/RedwoodGeneratorExtension.kt

```diff
@@ -18,10 +18,13 @@ package app.cash.redwood.gradle
 import org.gradle.api.artifacts.dsl.DependencyHandler
 import org.gradle.api.provider.Property
 
-public abstract class RedwoodGeneratorExtension : RedwoodSchemaExtension() {
+public abstract class RedwoodGeneratorExtension {
```

Comment:
Doesn't make sense for this to get existing `apiTracking` or the new `useFir` properties.